core/src/main/scala/org/apache/spark/sql/TiSparkSession.scala

```diff
@@ -238,7 +238,7 @@ object TiSparkSession {
     }
   }
 
-  def builder(): Builder = new Builder
+  def builder: Builder = new Builder
```

Comment:
I don't think this is where the modification takes place in this PR. 
If you have checked the `builder` function in `SparkSession` beforehand, you should find that `def builder()` is the correct way to define this Builder.
In contrast, you should modify parentheses in this line https://github.com/pingcap/tispark/blob/9c5063004534c9f76218b4ef7021fc2677eefce4/core/src/main/scala/org/apache/spark/sql/hive/thriftserver/TiSparkSQLEnv.scala#L50